packages/text-annotator/src/TextAnnotator.ts

```diff
@@ -30,8 +37,10 @@ export const createTextAnnotator = <E extends unknown = TextAnnotation>(
 
   const store: TextAnnotationStore = state.store;
 
-  const lifecycle = createLifecyleObserver<TextAnnotation, TextAnnotation | E>(store, selection, hover, viewport);
-  
+  const lifecycle = createLifecyleObserver<TextAnnotation, TextAnnotation | E>(store, selection, hover, viewport, opts.adapter, opts.autoSave);
```

Comment:
The latest Annotorious package version receives both the `adapter` and the `autoSave`